README.md

```diff
@@ -59,6 +59,7 @@ usage: waybar-module-pomodoro [options] [operation]
 
         --autow                     Starts a work cycle automatically after a break
         --autob                     Starts a break cycle automatically after work
+        --autor                     Restore timer to prior-state (even after shutdown)
```

Comment:
Maybe it could be called something else? `--cache`, `--state`, or something along those lines? `Persist timer state between sessions` for the help text maybe?

Reply:
Maybe,

--persist
Persist timer state between sessions?